mk/dev.mk

```diff
@@ -7,7 +7,7 @@ GIT_TAG = $(word 2, $(BUILD_INFO))
 GIT_COMMIT = $(word 3, $(BUILD_INFO))
 BUILD_DATE = $(word 4, $(BUILD_INFO))
 CI_TOOLS_VERSION = $(word 5, $(BUILD_INFO))
-ENVOY_VERSION ?= 1.30.3
+ENVOY_VERSION ?= 1.30.4
```

Comment:
It flaked with https://github.com/kumahq/kuma/actions/runs/9746235881/job/26897667259?pr=10693#step:13:13961 - is this new or something related to new Envoy?

Reply:
Is it a new flackiness? I see the second run was successful.

Reply:
This is known flake, we might need to tune/check it